api/src/backend/api/v1/views.py

```diff
@@ -509,10 +509,16 @@ def dispatch(self, request, organization_slug):
             return response
 
         extra = social_account.extra_data
-        user.first_name = extra.get("firstName", [""])[0]
-        user.last_name = extra.get("lastName", [""])[0]
-        user.company_name = extra.get("organization", [""])[0]
+        user.first_name = (
+            extra.get("firstName", [""])[0] if extra.get("firstName") else ""
+        )
+        user.last_name = extra.get("lastName", [""])[0] if extra.get("lastName") else ""
+        user.company_name = (
+            extra.get("organization", [""])[0] if extra.get("organization") else ""
+        )
         user.name = f"{user.first_name} {user.last_name}".strip()
+        if user.name == "":
+            user.name = "NoName"
```

Comment:
Does `NoName` come from any convention or something?

Reply:
No, we need a value because the name is mandatory. We can define another value

Reply:
Use `N/A` if possible then. For consistency with other wildcards.

Reply:
Maybe `Unnamed` sounds better

Reply:
Ok, we can change it